astro/src/diagrams/docs/lifecycle/authenticate-users/app-mfa-decision-tree.astro

```diff
@@ -0,0 +1,41 @@
+---
+import Diagram from 'src/components/mermaid/FlowchartDiagram.astro';
+
+const { alt } = Astro.props;
+
+//language=Mermaid
+const code = `
+graph TD
+    applicationMfaPolicy[What is MFA Policy At Application Level?] --> |No Application Policy Present| deferToTenant[Defer To The Tenant Policy]
+    applicationMfaPolicy --> |Disabled| noChallenge[Do Not Challenge User]
+    applicationMfaPolicy --> |Enabled| checkUserConfigEnabled[Check User Has MFA Method]
+    checkUserConfigEnabled --> |MFA Configured| contextualMFACheck[Check Request Context]
+    checkUserConfigEnabled --> |No MFA Configured| noChallengeNoMFAConfigured[Do Not Challenge User]
+    applicationMfaPolicy --> |Required| checkUserConfigRequired[Check User Has MFA Method]
+    checkUserConfigRequired --> |MFA Configured| contextualMFACheckRequired[Check Request Context]
+    checkUserConfigRequired --> |No MFA Configured| forceUserToSetupMFA[Prompt User To Set Up MFA]
+    contextualMFACheck --> |Context Check Fails| challenge[Challenge User]
+    contextualMFACheck --> |Context Check Succeeds| trustPolicyMFACheck[What is Application MFA Trust Policy?]
```

Comment:
I see that two branches converge here, but they could converge earlier at the `MFA Configured` step.

Reply:
I tried with a single node for `MFA configured` and it ended up looking really wonky.